Release notes for 1.23.x (#5249)
harness/developer-hub: release-notes/continuous-delivery.md

## January 2024

> ### Version 1.23.0

Reviewer: Suggest adding new heading for February and moving items under it.

#### Fixed Issues

> - Issue while selecting VPC if name is empty. VPC name is optional in AWS console but id is always there, but if the name is empty it is not getting selected. The issue is fixed now and instead of `vpc.name` it will get displayed as `vpc.id:vpc.name`. (CDS-89297)

Reviewer: ```suggestion
- Issue while selecting VPC if the name is empty. VPC name is optional in the AWS console but the id is always there, but if the name is empty it is not getting selected. The issue is fixed now and instead of `vpc.name` it will get displayed as `vpc.id:vpc.name`. (CDS-89297)
```

> - Application logs used to get printed in TAS deployment logs, the issue is fixed by introducing a new environment variable `DISABLE_CF_APP_LOG_STREAMING` and when it’s set to true it will redact all application logs, providing users with more flexibility in managing log visibility. (CDS-89172,ZD-56849)

Reviewer: ```suggestion
- Application logs used to get printed in TAS deployment logs, the issue is fixed by introducing a new environment variable `DISABLE_CF_APP_LOG_STREAMING` and when it’s set to true it will redact all application logs, providing users with more flexibility in managing log visibility. (CDS-89172, ZD-56849)
```

> - Parent pipeline has a chained pipeline stage and both the parent and child pipelines are in different organizations. While running the parent pipeline, the user group selection for the child approver user group displays parent pipeline organizations. The issue has now been fixed, and users should be able to select the correct user group(s) of the child pipeline that belong to different or the same organization.(CDS-89001)

Reviewer: ```suggestion
- Parent pipeline has a chained pipeline stage and both the parent and child pipelines are in different organizations. While running the parent pipeline, the user group selection for the child approver user group displays parent pipeline organizations. The issue has now been fixed, and users should be able to select the correct user group(s) of the child pipeline that belong to a different or the same organization. (CDS-89001, ZD-55502)
```

> - Approval message offset was not set properly, the issue is fixed now. (CDS-88839)

Reviewer: This one may have moved out of the release?  Not on the current list.

Reviewer 2: The NG_UI_SAAS version is listed as 1.6.0 which means it should be in the 1.23.xx release. Where are you seeing the current list of issues?

Reviewer: > The NG_UI_SAAS version is listed as 1.6.0 which means it should be in the 1.23.xx release. Where are you seeing the current list of issues?

https://harness.atlassian.net/wiki/spaces/PD/pages/21628289360/NG+Release+SAAS+January+29th+2024+for+1.23.x-+Build#Release-Notes-Eligible-Tickets

This is the page Akhil provides each release based on the Release Note Candidate set to Yes in Jiras. 

Here's the top level page: https://harness.atlassian.net/wiki/spaces/PD/pages/21596897283/Release+Notes+2024

> - When service was selected from **Projects** > **Service** > **Select Service** it used to open the configuration tab and when service was selected from **Deployments** > **Services** > **Select Service** it used to open the summary tab. The issue for this different behavior is now fixed and now via both ways users will be taken to the service's summary tab. (CDS-88692,ZD-56528)

Reviewer: ```suggestion
- When service was selected from **Projects** > **Service** > **Select Service** it used to open the configuration tab and when service was selected from **Deployments** > **Services** > **Select Service** it used to open the summary tab. The issue for this different behavior is now fixed and now via both ways, users will be taken to the service's summary tab. (CDS-88692, ZD-56528)
```

> - Jenkins `jobParameter` was getting added to the pipeline even if it was not a runtime input in the template. The issue is fixed now and `jobParameter` will not be added automatically. (CDS-88380)

Reviewer: ```suggestion
- Jenkins `jobParameter` was getting added to the pipeline even if it was not a runtime input in the template. The issue is fixed now and `jobParameter` will not be added automatically. (CDS-88380, ZD-55212)
```

> - Wrong target groups were getting attached when multiple load balancers were used with the same load balancer name. The issue is fixed now. (CDS-89615,ZD-55701)

Reviewer: Is this CDS-88229, not CDS-89615?

Reviewer 2: Looks like thats the case, seems that CDS-89615 is a hotfix ticket for CDS-88229 that was marked invalid.

> - When service is marked as runtime in a child pipeline and then configured from the chained pipeline inputs tab, the service input fields are left as runtime. The Trigger Input panel and Input-set form does not properly prompt for service input. This issue has been resolved. (CDS-87983)

Reviewer: ```suggestion
- When the service is marked as runtime in a child pipeline and then configured from the chained pipeline inputs tab, the service input fields are left as runtime. The Trigger Input panel and Input-set form do not properly prompt for service input. This issue has been resolved. (CDS-87983, ZD-55917)
```

> - WinRm connector was missing the back button. The issue is fixed now. (CDS-86773)

Reviewer 2: It was marked as not a release notes candidate, but this seems useful to put on this list.

> - There was a duplicate key error on stage execution info collection. The issue is fixed now. (CDS-85208)

Reviewer: Is this item still on the list?
